lighthouse-core/lib/page-functions.js

```diff
@@ -98,6 +98,13 @@ function getElementsInDocument(selector) {
       if (el.shadowRoot) {
         _findAllElements(el.shadowRoot.querySelectorAll('*'));
       }
+      // If the element has a contentWindow (IFrame), dig deeper.
+      // Try/Catch needed in case of cross-origin frame.
+      try {
+        if (el.contentWindow) {
+          _findAllElements(el.contentWindow.document.querySelectorAll('*'));
```

Comment:
is there something *slightly* more specific we could catch here? like testing the content window if its accesible? right now it'll silently drop all errors in the entire recursive call

Reply:
Figured out a better way to do this without the try/catch :), turns out that contentDocument is null, rather than throwing, for OOPIFs